airbyte-integrations/connectors/source-mysql/src/test/java/io/airbyte/integrations/source/mysql/CdcMysqlSourceTest.java

```diff
@@ -193,38 +187,16 @@ protected Database getDatabase() {
   }
 
   @Override
-  public void assertExpectedStateMessages(List<AirbyteStateMessage> stateMessages) {
-    for (AirbyteStateMessage stateMessage : stateMessages) {
+  public void assertExpectedStateMessages(final List<AirbyteStateMessage> stateMessages) {
+    for (final AirbyteStateMessage stateMessage : stateMessages) {
       assertNotNull(stateMessage.getData().get("cdc_state").get("state").get(MYSQL_CDC_OFFSET));
       assertNotNull(stateMessage.getData().get("cdc_state").get("state").get(MYSQL_DB_HISTORY));
     }
   }
 
   @Override
-  protected AirbyteCatalog expectedCatalogForDiscover() {
```

Comment:
why don't we need to override the catalog anymore? Not sure I understand how this change affects the output of `discover`

Reply:
Since for the new test we are adding a new table, this table is also discoverable now.